[docs] Update README.md (#876)
shopsys/shopsys: README.md

### Current State

Shopsys Framework is fully functional e-commerce platform with all basic functionality all e-commerce sites needs:
* Product catalogue
* Registered customers
* Basic orders management
* Back-end administration
* Front-end [full-text search](./docs/introduction/product-search-via-elasticsearch.md) and product filtering
* 3-step ordering process
* Basic CMS
* Support for several currencies, [languages, and domains](./docs/introduction/domain-multidomain-multilanguage.md)
* Full friendly URL for main entities
* Performance optimization through Elasticsearch, Redis, PostgreSQL
* Full core upgradability
* GDPR compliance
* Preparation for scalability

> * Manifest for orchestration via [Kubernetes](./docs/kubernetes/introduction-to-kubernetes.md) and [deployment to Google Cloud via Terraform](./docs/kubernetes/how-to-deploy-ssfw-to-google-cloud-platform.md)

Reviewer: ```suggestion
* Manifest for orchestration via [Kubernetes](./docs/kubernetes/introduction-to-kubernetes.md) 
* Support to easier [deployment to Google Cloud via Terraform](./docs/kubernetes/how-to-deploy-ssfw-to-google-cloud-platform.md)
```

### Plans for next releases

* Further performance optimizations

> * Modularization of pricing

Reviewer: ```suggestion

```

> * Modularization of pricing

Reviewer: ```suggestion
* Modularization of pricing
* Back-end API
```
